README.md

```diff
@@ -1,14 +1,28 @@
 # interview-app-ui
 
-Project descriotions
+Welcome to UI repository of Interview app. Goal of this project is to create a tool that can help technical interviewes to lead an interview with a increased confidence and ability to stay consistent accross of multiple interviews.
 
-### Scripts
+## Technica stack
+- React + TypeScript
+- AntD 
+
+## Local setup
+
+- run `npm install`
+- Make sure API is running on url specified in .env file
```

Comment:
`doc-link`, `spelling/grammar`

```suggestion
- Make sure [the API](https://github.com/3PillarGlobal-Czechia/interview-app-api) is running on the URL specified in the `.env` file
```

Reply:
`question`

@PlesnikJakub, do you think we should explain how to create and structure the `.env` file?

Or at least what the environment variable name is for the API endpoint? 🤔